Cython/Utility/CythonFunction.c

```diff
@@ -1792,13 +1795,54 @@ static PyObject* __Pyx_Method_ClassMethod(PyObject *method) {
         PyTypeObject *d_type = descr->d_common.d_type;
         #endif
         return PyDescr_NewClassMethod(d_type, descr->d_method);
+#else
+        return PyErr_Format(
```

Comment:
I suspect it would be possible to cover this case by setting `METH_CLASS` when the `PyMethodDef` is defined. I definitely don't think that's worth it for the first pass and may not be worth it in general.